tools/psa/README.md

```diff
@@ -0,0 +1,50 @@
+# PSA Tools
+
+This document will describe the following scripts:
+
+* \_\_init\_\_.py
+* generate_partition_code.py
+* mbed_spm_tfm_common.py
+* generate_mbed_spm_partition_code.py
+* generate_tfm_partition_code.py
+* release.py
+
+## \_\_init\_\_.py
+
+This file holds common function dedicated to help SiP with their post-build logic.
+
+* find_secure_image - Scans a Resource object to find the correct binary of the secure image to merge with the non-secure build.
+
+
+## Code generation scripts
+
+mbed-os holds 2 implementations of PSA:
+* MBED_SPM - Implementation for dual core v7 targets.
+* TF-M - Implementation for v8 targets.
+
+each implementation requires a set of auto generated files describing the secure partitions.
+
+* `generate_partition_code.py` - Generate files for both implementations.
+* `generate_mbed_spm_partition_code.py` - Generate files for BMED_SPM.
```

Comment:
I'd like to know what `BMED_SPM` is 😁

Reply:
![2vecn2](https://user-images.githubusercontent.com/17570086/53920197-04f49580-4075-11e9-82e4-c21c85471859.jpg)